From linux-usb-owner@vger.kernel.org Thu Apr 10 04:30:18 2008
From: Paulius Zaleckas <paulius.zaleckas@teltonika.lt>
Date: Thu, 10 Apr 2008 14:20:08 +0300
Subject: USB: oti6858: fix TCFLSH ioctl handling
To: linux-usb@vger.kernel.org
Message-ID: <ftkt6l$ru7$1@ger.gmane.org>


Removes unimplemented TCFLSH handling from oti6858, because it was 
preventing TCFLSH handling by upper layer (line discipline) drivers (see 
drivers/char/tty_io.c line 3450).

Signed-off-by: Paulius Zaleckas <paulius.zaleckas@teltonika.lt>
Acked-by: Alan Cox <alan@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/oti6858.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/usb/serial/oti6858.c
+++ b/drivers/usb/serial/oti6858.c
@@ -814,10 +814,6 @@ static int oti6858_ioctl(struct usb_seri
 				__FUNCTION__, port->number, cmd, arg);
 
 	switch (cmd) {
-		case TCFLSH:
-			/* FIXME */
-			return 0;
-
 		case TIOCMBIS:
 			if (copy_from_user(&x, user_arg, sizeof(x)))
 				return -EFAULT;
